### PR TITLE
Add optional "Advertisement" label to Fabrics

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Yeah!

picture 1
picture 2

### CODE REVIEW

File: src/_shared/scss/_core.scss [hunk 1]
```diff
@@ -127,6 +127,25 @@ $mq-breakpoints: (
     display: block;
 }
 
+.creative__label {
+    background-color: #f6f6f6;
```
Comment (JonNorman) [inline src/_shared/scss/_core.scss]: Can you use `$neutral-8` instead?

File: src/fabric-video/index.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 .creative--fabric-video {
     margin: auto;
+    max-width: 1920px;
```
Comment (JonNorman) [inline src/fabric-video/index.scss]: Would we benefit from storing this in a variable somewhere, e.g. `ultimate-super-wide-width-width`? :)

File: src/_shared/scss/_core.scss (hunk 1 above)
Comment (regiskuckaertz) [inline src/_shared/scss/_core.scss]: Yes, good call!

File: src/fabric-video/index.scss (hunk 2 above)
Comment (regiskuckaertz) [inline src/fabric-video/index.scss]: Yeah I was actually surprised this limit only applied to Fabric Video. I might use it elsewhere, so will `var` the hell out of it 😆

**Comment by regiskuckaertz** [comment]
[Fabrics: add option ad label](https://trello.com/c/R8CFCcnx/120-fabrics-add-option-ad-label)

**Pull request merged**